examples/hashes/connector.py

```diff
@@ -0,0 +1,92 @@
+# This example shows how to calculate hash of fields to be used as primary key. This is useful in scenarios where the
+# incoming rows do not have any field suitable to be used as a Primary Key.
+# Fivetran recommends to define primary keys (https://fivetran.com/docs/connectors/connector-sdk/best-practices#declaringprimarykeys) when writing a connector.
+# This is to avoid disruptions caused due to a schema change (addition/removal of a column, etc) that might be needed in the maintainance phase of your connector.
+# See the Technical Reference documentation (https://fivetran.com/docs/connectors/connector-sdk/technical-reference#update)
+# and the Best Practices documentation (https://fivetran.com/docs/connectors/connector-sdk/best-practices) for details.
+
+# Import required classes from fivetran_connector_sdk
+from fivetran_connector_sdk import Connector
+from fivetran_connector_sdk import Operations as op
+
+# Import built-in Python modules
+import hashlib
+import json
+
+# Define the schema function which lets you configure the schema your connector delivers.
+# See the technical reference documentation for more details on the schema function
+# https://fivetran.com/docs/connectors/connector-sdk/technical-reference#schema
+# The schema function takes one parameter:
+# - configuration: a dictionary that holds the configuration settings for the connector.
+# In this example, there is no use of the configuration
+def schema(configuration: dict):
+    return [
+        {
+            "table": "user",  # Name of the table in the destination.
+            "primary_key": ["_fivetran_id"],  # Primary key column(s) for the table. Should we change the name?
+            "columns": {  # Define the columns and their data types.
+                "first_name": "STRING",  # String column for the first name.
+                "last_name": "STRING",  # String column for the last name.
+                "email": "STRING",  # String column for the email.
+                "updated_at": "UTC_DATETIME",  # UTC date-time column for the updated_at.
+            },
+        }
+    ]
+
+
+# Define the update function, which is a required function, and is called by Fivetran during each sync.
+# See the technical reference documentation for more details on the update function
+# https://fivetran.com/docs/connectors/connector-sdk/technical-reference#update
+# The function takes two parameters:
+# - configuration: dictionary contains any secrets or payloads you configure when deploying the connector
+# - state: a dictionary contains whatever state you have chosen to checkpoint during the prior sync
+# The state dictionary is empty for the first sync or for any full re-sync
+def update(configuration: dict, state: dict):
+
+    # Represents a record fetched from source
+    row = {
```

Comment:
We need some more records here and also some interesting records where some of the fields, like email, are empty so that customers can get a better idea.
The whole idea of hash should be visible where customer cannot use composite PK.

Reply:
Added